README.md

```diff
@@ -118,17 +124,62 @@ error = p.show(p.add(p.len(p.int(5)), p.int(3)))
 print(error)   # Output: len(5)+3 contains ill-typed subterm len(5)
 ```
 
+### JavaScript/TypeScript
+
+```javascript
+import pyfinalo from 'pyfinalo_js';
+
+// Direct evaluation
+const result = pyfinalo.show(pyfinalo.add(pyfinalo.int(5), pyfinalo.int(3)));
+console.log(result); // Output: "8 : int"
+
+// String operations
+const strResult = pyfinalo.show(pyfinalo.len(pyfinalo.str("hello")));
+console.log(strResult); // Output: "5 : int"
+
+// Complex expressions
+const complex = pyfinalo.show(
+  pyfinalo.add(
+    pyfinalo.mul(pyfinalo.int(2), pyfinalo.int(3)),
+    pyfinalo.int(4)
+  )
+);
+console.log(complex); // Output: "10 : int"
+```
+
 ## Available Interpreters
 
+### Python Modules
 - **`pyfinalo`**: Direct value interpreter (`DirectValInterp`) - evaluates expressions immediately
 - **`pyfinalo_ast`**: AST interpreter (`UntypedAstInterp`) - generates abstract syntax trees
 - **`pyfinalo_explain`**: Explanatory interpreter (`ExplainInterpUntyped`) - provides detailed type checking
 
+### JavaScript/TypeScript Module
+- **`pyfinalo_js`**: Direct value interpreter available as an npm package
+
 ## Development
 
+### Building JavaScript Bindings
+
+```bash
+# Install dependencies
+cd src/lib_pyfinalo_js
+bun install
+
+# Build the JavaScript module
+bun run build
+
+# Run JavaScript tests
+bun test
+```
+
 ### Testing
 ```bash
+# OCaml tests
```

Comment:
mention that this also run python tests (i.e. cram tests under lib_pyfinalo_js/test)